### PR TITLE
Add more lints to correspond with existing build system

### PR DESCRIPTION
**Pull request by pawelchcki**
# What does this PR do?

A brief description of the change being made with this pull request.

# Motivation

What inspired you to submit this pull request?

# Additional Notes

Anything else we should know when reviewing?

# How to test the change?

Describe here in detail how the change can be validated.

### CODE REVIEW

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -15,21 +15,60 @@ jobs:
...
+        run: > 
+          cargo bundle-licenses \
+            --format yaml \
+            --output /tmp/CI.yaml \
+            --previous LICENSE-3rdparty.yml \
+            --check-previous 
+
+    
+    
+
```
Comment (paullegranddc) [inline .github/workflows/lint.yml:74]:
```suggestion

```

File: LICENSE-3rdparty.yml [hunk 2]
```diff
@@ -777,10 +967,16 @@ third_party_libraries:
...
   - package_name: webpki
     package_version: 0.22.0
-    license: License specified in file (/tmp/dd-cargo/registry/src/github.com-1ecc6299db9ec823/webpki-0.22.0/LICENSE)
+    license: License specified in file (/Users/paul.legranddescloizeaux/.cargo/registry/src/github.com-1ecc6299db9ec823/webpki-0.22.0/LICENSE)
     licenses:
-      - license: License specified in file (/tmp/dd-cargo/registry/src/github.com-1ecc6299db9ec823/webpki-0.22.0/LICENSE)
+      - license: License specified in file (/Users/paul.legranddescloizeaux/.cargo/registry/src/github.com-1ecc6299db9ec823/webpki-0.22.0/LICENSE)
```
Comment (paullegranddc) [inline LICENSE-3rdparty.yml:972]: Oh I messed up here when recreating the license file

Comment (pawelchcki) [reply LICENSE-3rdparty.yml:972]: no worries, BTW I was looking for some other ways to handle the licenses. And so far haven't found any good/better alternative

**Review by paullegranddc — APPROVED** [review]

File: .github/workflows/lint.yml [hunk 3]
```diff
@@ -67,4 +67,10 @@ jobs:
...
             --previous LICENSE-3rdparty.yml \
-            --check-previous 
\ No newline at end of file
+            --check-previous 
+      - name: export the generated license file on failure
+        if: ${{ failure() }}
+        uses: actions/upload-artifact@v1
+        with:
+          name: LICENSE-3rdparty.yml
+          path: /tmp/CI.yaml
```
Comment (pawelchcki) [inline .github/workflows/lint.yml:76]: ❤️

**Review by morrisonlevi — APPROVED** [review]

**Pull request merged**